cnf-certification-test/identifiers/identifiers.go

```diff
@@ -362,6 +362,10 @@ var (
 		Url:     formTestURL(common.LifecycleTestKey, "persistent-volume-reclaim-policy"),
 		Version: versionOne,
 	}
+	TestContainersImageTag = claim.Identifier{
+		Url:     formTestURL(common.LifecycleTestKey, "test-Containers-Image-Tag"),
```

Comment:
```suggestion
		Url:     formTestURL(common.LifecycleTestKey, "test-Containers-image-Tag"),
```

No need for capitalization.